docs/settingup.md

```diff
@@ -436,9 +436,34 @@ Clear all selections in an app.
 
 ## ClickActionButton action
 
-Simulate clicking an action-button
-
-
+A `ClickActionButton`-action simulates clicking an _action-button_. An _action-button_ is a chart item which, when clicked, executes a series of actions. The series of actions contained by an action-button begins with any number _generic button-actions_ and ends with an optional _navigation button-action_.
+
+**Note:** Clicking an action-button may have side effects such as changing sheet and locking selections, which highly affect the outcome of following actions.
```

Comment:
I'm not sure if we need this disclaimer. Many of the actions affect the state of the app and thus any succeeding actions, but we don't have any such note on those. I think this may mainly be an internal concern for the app structure functionality since it leaves the app in an undetermined state.

Reply:
This could surely be noted in most actions, but since the button contain a "hidden" macro, the script-authors may have some extra need of a heads up.

Reply:
But as you say, the app-state should be kept to give appropriate warnings :)